### ISSUE TITLE
Crash on Asyncdisplaykit 2.0

### ISSUE THREAD

**Issue by dungnc** [ide/WebASDKImageManager#7]
For AS 2.0 a method in ASImageDownloaderProtocol has changed:

`- (nullable id)downloadImageWithURL:(NSURL *)URL
                      callbackQueue:(dispatch_queue_t)callbackQueue
                   downloadProgress:(nullable ASImageDownloaderProgress)downloadProgress
                         completion:(ASImageDownloaderCompletion)completion;`

please update it on "SDWebASDKImageManager"
Thanks!

**Comment by ide** [comment]
Please submit a PR and test plan if you need this.

**Comment by skensell** [comment]
I've opened a PR for this: https://github.com/ide/WebASDKImageManager/pull/8

**Comment by skensell** [comment]
Until the PR is merged you can use my branch like this from your Podfile:

```
  pod 'AsyncDisplayKit/Core', '>= 2.0'
  pod 'WebASDKImageManager', :git => 'https://github.com/skensell/WebASDKImageManager.git', :branch => 'Async2'
```

**Comment by ide** [comment]
Published 2.0 to CocoaPods.

**Comment by skensell** [comment]
Thanks!
